src/AzureRMTools.ts

```diff
@@ -209,20 +213,68 @@ export class AzureRMTools {
                 diagnostics.push(this.getVSCodeDiagnosticFromIssue(deploymentTemplate, warning, vscode.DiagnosticSeverity.Warning));
             }
 
-            if (ext.addCompletionDiagnostic) {
-                // Add a diagnostic to indicate expression validation is done (for testing)
-                diagnostics.push(<vscode.Diagnostic>{
-                    severity: vscode.DiagnosticSeverity.Information,
-                    message: expressionsDiagnosticsCompletionMessage,
-                    source: expressionsDiagnosticsSource,
-                    code: ""
-                });
+            let languageServerLanguageDiagnostic = this.getLanguageServerStateDiagnostic();
```

Comment:
`const` here and several other places in the PR.

Reply:
Personally I'm not a fan of that rule, it's annoying when you can't change values during debugging sessions.  If I reconsider I'll turn on the lint rule.